caluma/caluma_form/serializers.py

```diff
@@ -243,7 +245,7 @@ def validate(self, data):
         return super().validate(data)
 
     class Meta(SaveQuestionSerializer.Meta):
-        fields = SaveQuestionSerializer.Meta.fields
+        fields = SaveQuestionSerializer.Meta.fields + ["hint_text"]
```

Comment:
Should probably be moved to `SaveQuestionSerializer.Meta`, so you won't have to explicitly extend the fields list in every subtype

Reply:
Ah, nevermind, needs to be this way because form, static and action buttons don't have hints